src/services/FirebaseAdminService.ts

```diff
@@ -0,0 +1,20 @@
+import admin from 'firebase-admin';
+import '@firebase/auth';
+
```

Comment:
can we make this a class?

Reply:
I also think it'd be nice to call this AuthService or AccountService, that way it makes sense if we later don't use firebase anymore

Reply:
I'll call it SignupService. How does that sound?

Reply:
I think in general it's better to name things a little more generic - later on we can expose deleteAccount, setVerifyEmail to write scripts and maybe manage accounts. Let's go with accountservice?

Reply:
Yep, changed.